### DISCUSSION TITLE
BabyPod - A remote control for Baby Buddy for new parents

### DISCUSSION THREAD

**Discussion by skjdghsdjgsdj** [babybuddy/babybuddy#819] [Show and tell]
I wanted to show off a device I made that integrates with Baby Buddy called BabyPod:

Hardware: https://github.com/skjdghsdjgsdj/babypod-hardware
Software (CircuitPython): https://github.com/skjdghsdjgsdj/babypod-software
Printables.com: https://www.printables.com/model/872095-babypod-a-remote-control-for-baby-buddy-for-new-pa

My partner and I have been using this device--and recently printed another for convenience's sake--to help track our newborn's behavior. I thought you might want to add it to the `README.md` for Baby Buddy as an integration project that third parties have built. It's been immensely helpful as a near-one-click way of tracking feedings and other activies.

I released the hardware design and the software to GitHub for others to contribute to. I have no doubt there are bugs and things to be improved.

**Comment by cdubz** [comment]
Very cool!! Thanks for sharing. Will add this to BBotW 😄
